server/player.lua

```diff
@@ -37,15 +37,14 @@ function Login(source, citizenid, newData)
         return false
     end
 
-    local license = GetPlayerIdentifierByType(source --[[@as string]], 'license2') or GetPlayerIdentifierByType(source --[[@as string]], 'license')
-    local userId = storage.fetchUserByIdentifier(license)
-
+    local license, license2 = GetPlayerIdentifierByType(source --[[@as string]], 'license'), GetPlayerIdentifierByType(source --[[@as string]], 'license2')
+    local userId = storage.fetchUserByIdentifier(license2) or storage.fetchUserByIdentifier(license)
+    assert(userId ~= nil, ('User does not exist. Licenses checked %s, %s'):format(license2, license))
```

Comment:
I think we should use `lib.print.error` and return false rather than throwing an error, similarly to what we do when validating the `source` variable.

Reply:
done